6_visualize/src/prep_storm_point_fun.R

```diff
@@ -1,6 +1,7 @@
 
-prep_storm_point_fun <- function(storm_points_ind_file, DateTime, hurricane_col){
-  storm_points_sf <- readRDS(sc_retrieve(storm_points_ind_file))
+fetch_read <- function(ind_file) readRDS(sc_retrieve(ind_file))
```

Comment:
Theoretically, using this function will allow us to keep an RDS file full of time steps in memory while all the pngs draw?

Reply:
That should be the case, yes. [remake](https://github.com/richfitz/remake) is backed by [storr](https://github.com/richfitz/storr), which provides "cached in-memory copies" of objects in the storr.